octokit/client.py

```diff
@@ -9,11 +9,12 @@
 
 # https://code.google.com/p/uri-templates/wiki/Implementations
 
+from .rate_limit import RateLimit, _RateLimit
 from .resources import Resource
-from .session import Session
 
+import requests
 
-class Client(Resource):
+class Client(Resource, RateLimit):
```

Comment:
I think this is reverse `Client(RateLimit, Resource)`. Otherwise, Client is a RateLimit instead of a resource